src/Momento.Sdk/Momento.Sdk.csproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
 		<!-- Build Configuration -->
-		<TargetFramework>netstandard2.0</TargetFramework>
+		<TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
@malandis I went ahead and did this now even though I haven't yet introduced the stuff that requires it.